### ISSUE TITLE
installLisk.sh  not working on High Sierra 

### ISSUE THREAD

**Issue by RachBLondon** [LiskHQ/lisk-build#312]
I have been trying to run the binary test node on OS `10.13.4` and I keep running in to the following issue:

```
Coldstarting Lisk for the first time
√ Postgresql started successfully.
X Failed to create Postgresql user.
Installation failed. Cleaning up...
```

@Nazgolze spent some time looking at this yesterday.

**Comment by ghost** [comment]
@RachBLondon can you please provide some more info:
 - what version are you trying to use
 - command you're using
 - whether you've got postgres running locally

**Comment by ghost** [comment]
Looks like the binaries we ship are linked against postgresql 10 but we provide 9.6 libraries; oniguruma is also missing (or rather should be explicitely disabled at runtime)

**Comment by ghost** [comment]
This issue will be solved by #314 (getting rid of postgres 10) and #316

**Comment by ghost** [comment]
Won't fix: #317
